module/VuFindAdmin/src/VuFindAdmin/Controller/FeedbackController.php

```diff
@@ -0,0 +1,306 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Class FeedbackController
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Moravian Library 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  VuFindAdmin\Controller
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+namespace VuFindAdmin\Controller;
+
+use Laminas\Db\Sql\Select;
+use VuFind\Db\Table\Feedback;
+
+/**
+ * Class FeedbackController
+ *
+ * @category VuFind
+ * @package  VuFindAdmin\Controller
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+class FeedbackController extends AbstractAdmin
+{
+    /**
+     * Params
+     *
+     * @var array
+     */
+    protected $params;
+
+    /**
+     * Get the url parameters
+     *
+     * @param string $param A key to check the url params for
+     *
+     * @return string
+     */
+    protected function getParam($param)
```

Comment:
I realize that this controller is derived from TagsController and that's why this design is being used... but TagsController is a bit confusing, and maybe we don't want to duplicate all of its flaws. It's confusing to me that we assign values to $this->params in each of the actions just so that we control whether GET or POST is prioritized... and that we do redundant parameter lookups regardless. What if we eliminated the $params property entirely and changed getParam to something like this?

```
protected function getParam($param, $prioritizePost = false)
{
    $primary = $prioritizePost ? 'fromPost' : 'fromQuery';
    $secondary = $prioritizePost ? 'fromQuery' : 'fromPost';
    return $this->params()->$primary($param) ?? $this->params()->$secondary($param);
}
```

Then we could modify `deleteAction()` / `updateStatusAction()` to pass `true` as the second parameter in all the `getParam` calls, and all the assignments to $this->params could be eliminated.

Reply:
If you agree with this approach, maybe I should open a separate PR to make the same changes in TagsController for consistency.